docs/CHANGELOG.rst

```diff
@@ -6,6 +6,14 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+2023-11-16
+==========
+
+- Bump the version of ``lxml`` Python package to ``4.9.3`` in 
```

Comment:
In the past, we've used `packagename==version` notation. Perhaps something along the lines of

> Bump Python package to `lxml==4.9.3` in `resolwebio/common:4.1.1`